### PR TITLE
update pricing table name and cols

### PR DESCRIPTION
**Pull request by tarikceric**
1. Adds new models and sources for the prices v2 upgraded models in crosschain
2. Updates models referencing v1 silver price models
  -for models updated in `models/silver/liquidity_pool`, confirmed that values from original metadata table match new metadata table 
3. Add deprecation notices

### CODE REVIEW

File: models/silver/prices/silver__token_prices_coin_market_cap_hourly.sql [hunk 1]
```diff
@@ -78,8 +78,8 @@ base_legacy_prices AS (
         price AS CLOSE
     FROM
         {{ source(
-            'shared',
-            'prices_v2'
```
Comment (desmond-hui) [inline models/silver/prices/silver__token_prices_coin_market_cap_hourly.sql:82]: The old source should be removed from `sources.yml` as well?

File: models/gold/price/price__fact_hourly_token_prices.sql [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        inserted_timestamp,
+        '2000-01-01'
+    ) AS inserted_timestamp,
+    COALESCE(
+        modified_timestamp,
+        '2000-01-01'
+    ) AS modified_timestamp
+FROM
+    {{ ref('silver__token_prices_coin_market_cap_hourly') }}
+UNION
```
Comment (desmond-hui) [inline models/gold/price/price__fact_hourly_token_prices.sql:30]: Should this be a `UNION ALL`?

File: models/silver/prices/silver__token_prices_coin_market_cap_hourly.sql [hunk 3]
```diff
@@ -78,8 +78,8 @@ base_legacy_prices AS (
...
+            'crosschain_bronze',
+            'legacy_prices'
```
Comment (desmond-hui) [inline models/silver/prices/silver__token_prices_coin_market_cap_hourly.sql:82]: This source does not exist in `sources.yml`

**Review by desmond-hui — CHANGES_REQUESTED** [review]

File: models/gold/price/price__fact_hourly_token_prices.sql (hunk 2 above)
Comment (tarikceric) [reply models/gold/price/price__fact_hourly_token_prices.sql]: updated

File: models/silver/prices/silver__token_prices_coin_market_cap_hourly.sql (hunk 1 above)
Comment (tarikceric) [reply models/silver/prices/silver__token_prices_coin_market_cap_hourly.sql:82]: yup old source can be removed since not used anywhere - updated

**Review by desmond-hui — APPROVED** [review]

**Review by desmond-hui — APPROVED** [review]

**Review by eric-laurello — APPROVED** [review]

**Pull request merged**